index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+
+  <meta charset="utf-8">
+  <title>Trekkin'</title>
+
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/foundation/6.2.4/foundation.min.css">
+  <link href="trek.css" media="screen" rel="stylesheet" type="text/css">
+    <script src="https://code.jquery.com/jquery-3.2.1.min.js"
+    integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4="
+    crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/foundation/6.2.4/foundation.min.js"></script>
+    <script type="text/javascript" src="trek.js"></script>
+
+  </head>
+
+  <body>
+
+    <button id="load"> Travel! </button>
+    <div id="tripinfo"><ul></ul></div>
+    <div id="location"></div>
+    <div id="fail"></div>
```

Comment:
Maybe moving this `div` element to the top would be preferable, that way it will show at the top of the app and can't  be missed.